god-care-api/src/main/java/com/godcare/api/controller/ProductController.java

```diff
@@ -0,0 +1,71 @@
+package com.godcare.api.controller;
+
+import com.godcare.api.service.ProductService;
+import com.godcare.common.dto.ResisterProductRequest;
+import com.godcare.api.util.ApiUtils;
+import com.godcare.api.vo.Response;
+import com.godcare.api.entity.Product;
+import com.godcare.common.dto.UpdateProductRequest;
+import com.godcare.common.dto.ViewProductResponse;
+import io.swagger.annotations.Api;
+import io.swagger.v3.oas.annotations.Operation;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.*;
+
+
+@RestController
+@RequestMapping("/api/v1/products")
+@RequiredArgsConstructor
+@Api(tags = "상품 API")
+public class ProductController<V> {
+
+    private final ProductService productService;
+
+    @Operation(summary = "상품 등록 API ", description = "새로운 상품을 업로드하는 API")
+    @PostMapping(path = "")
+    public ResponseEntity<Response<Object>> addProduct(@RequestBody ResisterProductRequest resisterProductRequest) {
```

Comment:
```suggestion
    @ResponseStatus(HttpStatus.Created)
    public ResponseEntity<Response<Object>> addProduct(@RequestBody ResisterProductRequest resisterProductRequest) {
```

https://www.baeldung.com/spring-response-status